src/components/DropdownItem.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+
+function DropdownItem(props) {
+    // function onClick(event) {
+    //     //Set the date to the item value
+    //     props.setDate(props.item);
+    const onClick = () => props.visitItem(props.item)
```

Comment:
I know this doesn't feel that helpful, but now we are free to change the implementation inside of `visitItem` inside of our App.js and this component will never know. That makes this component dumb, and that is good.